frontend/src/app/store/reducers/cases.ts

```diff
@@ -74,13 +76,13 @@ export const casesSlice = createSlice({
     },
     setCaseFile: (state, action) => {
       const {
-        payload: { caseIdentifier, note, equipment, subject, reviewComplete, decision },
+        payload: { caseIdentifier, note, equipment, subject, reviewComplete, decision, authorization },
```

Comment:
Does this need a redux Migration because you are changing the shape of the payload?

Reply:
In this case no. There's a couple reasons for this, the first being that the case property isn't in the redux-persist whitelist, but even if it was the authorization property is `authorization?: PermitSite` so even if the case was persisted it wouldn't cause an error during rehydration because of the ? undefined operator on the property.